src/app/frontend/replicationcontrollerdetail/serviceendpoint_directive.js

```diff
@@ -22,6 +22,7 @@ export default function serviceEndpointDirective() {
     templateUrl: 'replicationcontrollerdetail/serviceendpoint.html',
     scope: {
       'endpoint': '=',
+      'url': '@',
```

Comment:
When url param is '@' you'll show it as a link when the value is `foo` `true` `false` or `x`. How about changing it to `=`?